eventbridge/platform/tf-aws/eventbridge.w

```diff
@@ -0,0 +1,85 @@
+bring cloud;
+bring aws;
+bring "./../../types.w" as types;
+bring "cdktf" as cdktf;
+bring "./bus.w" as bus;
+bring "@cdktf/provider-aws" as tfAws;
+
+pub class EventBridge impl types.IEventBridge {
```

Comment:
I feel this class should be called `Bus` (and then it will be referred as `eventbridge.Bus` and I am not sure why we need the separate bus type and "instance". Seems a bit convoluted.